etna/Dockerfile

```diff
@@ -2,3 +2,5 @@ FROM etna-base-dev
 ENV ETNA_GEM_DEVELOPMENT=1
 COPY . /etna
 RUN rm -rf /app && ln -sfT /etna /app
+
+RUN cd /etna && npm run build
```

Comment:
My attempt to compile the TypeScript files in `etna-js` into the image, so any imports in the other apps can work.

Reply:
:+1: 
Ultimately, a "single package.json / webpack.js" setup will look like this, too -- build into etna-base-dev, and share that out to other images.